pkg/common/cns-lib/volume/manager.go

```diff
@@ -155,8 +155,13 @@ func (m *volumeManager) CreateVolume(spec *cnstypes.CnsVolumeCreateSpec) (*cnsty
 			klog.Errorf("Failed to get taskInfo for CreateVolume task from vCenter %q with err: %v", m.virtualCenter.Config.Host, err)
 			return nil, err
 		}
-		// Store the taskInfo details and taskInfo object expiration time in volumeTaskMap
-		volumeTaskMap[spec.Name] = createVolumeTaskDetails{taskInfo, time.Now().Add(time.Hour * time.Duration(defaultOpsExpirationTimeInHours))}
+		blockBackingDetails, ok := spec.BackingObjectDetails.(*cnstypes.CnsBlockBackingDetails)
+		// There is no need add task details into volumeTaskMap for static volume provisioning
```

Comment:
What happens if we add the task details? Any side effects? If so, can you call this in the PR description?

Reply:
Added.